### PR TITLE
Vinay Work

### PR DESCRIPTION
**Pull request by vinaykathiria**

### CODE REVIEW

File: Vinay/palindrome.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+s = input("Enter the sentence:")
+if s == s.replace("",""):
```
Comment (Patibandha) [inline Vinay/palindrome.py:2]: what about other special characters?

**Review by Patibandha — COMMENTED** [review]
also include assignment #3 and test cases for all three asignmen s

File: Vinay/palindrome.py (hunk 1 above)
Comment (vinaykathiria) [reply Vinay/palindrome.py:2]: I ran the code with A man, a plan, a canal: Panama$ it is showing me current output. So I think it is working.

**Review by Patibandha — CHANGES_REQUESTED** [review]
add test cases for all your program

**Pull request merged**